packages/login/redux/network/actions.js

```diff
@@ -19,14 +19,20 @@ import bip39 from 'bip39'
 import Accounts from 'web3-eth-accounts'
 import { login } from '@chronobank/core/redux/session/actions'
 import { stopSubmit, SubmissionError } from 'redux-form'
-import { push } from 'react-router-redux'
+import { push, goBack } from 'react-router-redux'
```

Comment:
**[MAJOR]**

@chronobank/login must not depend on web-related things.
Please replace this import to import from appropriate @chronobank/core-dependencies
(in ChronoMint-RN we will provide our own empty push and goBack).

We can't even build with current implementation.